### ISSUE TITLE
Only scraping messages?

### ISSUE THREAD

**Issue by ProjektPremium** [DIGITALCRIMINAL/OnlyFans#495]
Updated to Python 3.9, updated the Scraper. Now, run it with a working config files, upon logging in after selecting 0 for OnlyFans it just scrapes messages. Can't choose who or what to scrape. Even messages from people who I'm no longer subscribed to, but only the paid for content.

**Comment by DIGITALCRIMINAL** [comment]
The script scrapes paid content that's on the account. Read #jobs on the readme.

**Comment by ProjektPremium** [comment]
Yes, I have checked that. It doesn't give me a list of names of who to scrape, it just automatically goes through messages and scrapes the paid for files. Doesn't scrape the remainder of the content. Doesn't even give the option to choose who to scrape.

**Comment by ProjektPremium** [comment]
https://ibb.co/rGgCR5p

**Comment by DIGITALCRIMINAL** [comment]
Are you using more than one account

**Comment by ProjektPremium** [comment]
No.

These are my settings;

      "settings": {
        "auto_choice": "",
        "auto_scrape_names": false,
        "auto_scrape_apis": true,
        "jobs": {
          "scrape_names": true,
          "scrape_paid_content": true
        },

**Comment by DIGITALCRIMINAL** [comment]
Okay then just set scrape paid content to false if you want to skip scraping paid content.

**Comment by ProjektPremium** [comment]
I'll try that. 

Even removed;
       "jobs": {
          "scrape_names": true,
          "scrape_paid_content": true
        },

**Comment by DIGITALCRIMINAL** [comment]
Don't remove that... it will just regenerate the whole thing again.

**Comment by ProjektPremium** [comment]
Yeah I noticed that, set it to false, now it's showing the names list. I'm going to scrape somebody and get back to you.

**Comment by ProjektPremium** [comment]
I think it runs fine for everybody else because they don't pay for things lol.

**Comment by DIGITALCRIMINAL** [comment]
The script was working as intended; it just scrapes paid content first then it scrapes normal content

**Comment by ProjektPremium** [comment]
Okay, here are the results. I set "scrape_paid_content" to false, it gets me the list of subbed names. 

Get to these options:
Scrape: a = Everything | b = Images | c = Videos | d = Audios | e = Texts
a

Scrapes everything, even the paid for stuff.

When I set "scrape_paid_content" to "true" it goes straight to my messages, and downloads everything that's been paid for, even people who I haven't been subscribed to for months, once it does all the PMs it stops the scrape as if there is nothing else for it to do.

I will leave it with the settings the way they are now that they are working.

Thank you again for your time. Check your emails once in a while. ;)
